### ISSUE TITLE
Keyboard doesn't show up Android 12

### ISSUE THREAD

**Issue by sirjofri** [echoline/drawterm-android#10]
After starting a drawterm session and clicking the KB button the keyboard doesn't show up on my device (pixel 4). I first have to switch to another app (or go back to home screen), then switch back and click the button again.

**Comment by echoline** [comment]
so many bugs... :(

**Comment by sirjofri** [comment]
Yeah, I know. It's very sad and I'd like to help more, but I have basically no experience with android and I also don't like the platform for programming...
